Extensions/dot/mcclim-dot.asd

```diff
@@ -0,0 +1,37 @@
+(in-package #:asdf-user)
+
+(defsystem "mcclim-dot"
+  :license "MIT"
+  :description "Support for using DOT based graph layout engines."
+  :depends-on ("mcclim-dot/core" "mcclim-dot/graphviz")
+  :in-order-to ((test-op (test-op "mcclim-dot/test"))))
+
+(defsystem "mcclim-dot/core"
+  :license "MIT"
+  :description "Core DOT routines."
+  :depends-on ("mcclim"
+               (:version "cl-dot" "0.9")
```

Comment:
if I understand this correctly, all needed symbols are available in the library; perhaps to not depend on a version that is not yet available in ql, this commit (c4e486f5a611dda6ff5dd90aa1f3f5726fe00821) should be reverted? I don't have a strong opinion about this.